File: OPENAPI-REST-API/swagger-client/dart/lib/model/vps_plesk12_data.dart
part of swagger.api;

class VpsPlesk12Data {
  
  VpsPleskLicense admin = null;

  VpsPleskLicense pro = null;

  VpsPleskLicense host = null;

  VpsPlesk12Data();

  @override
  String toString() {
    return 'VpsPlesk12Data[admin=$admin, pro=$pro, host=$host, ]';
  }

  VpsPlesk12Data.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    admin = new VpsPleskLicense.fromJson(json['admin']);
    pro = new VpsPleskLicense.fromJson(json['pro']);
    host = new VpsPleskLicense.fromJson(json['host']);
  }

  Map<String, dynamic> toJson() {
    return {
      'admin': admin,
      'pro': pro,
      'host': host
     };
  }

  static List<VpsPlesk12Data> listFromJson(List<dynamic> json) {
    return json == null ? new List<VpsPlesk12Data>() : json.map((value) => new VpsPlesk12Data.fromJson(value)).toList();
  }

  static Map<String, VpsPlesk12Data> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, VpsPlesk12Data>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new VpsPlesk12Data.fromJson(value));
    }
    return map;
  }
}

File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/VpsPlesk12Data.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class VpsPlesk12Data {
    /// <summary>
    /// Gets or Sets Admin
    /// </summary>
    [DataMember(Name="admin", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "admin")]
    public VpsPleskLicense Admin { get; set; }

    /// <summary>
    /// Gets or Sets Pro
    /// </summary>
    [DataMember(Name="pro", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "pro")]
    public VpsPleskLicense Pro { get; set; }

    /// <summary>
    /// Gets or Sets Host
    /// </summary>
    [DataMember(Name="host", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "host")]
    public VpsPleskLicense Host { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class VpsPlesk12Data {\n");
      sb.Append("  Admin: ").Append(Admin).Append("\n");
      sb.Append("  Pro: ").Append(Pro).Append("\n");
      sb.Append("  Host: ").Append(Host).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
